### PR TITLE
VIDEO-4419 | Update isSupported to match browser matrix support

### PR DESCRIPTION
**Pull request by charliesantos**
Previously, we don't check for mobile browser specifically, we only test for WebRTC support. Now that mobile browsers are starting to support WebRTC, isSupported is returning true for these browsers. We want to still return false for these until QE officially support them.

### CODE REVIEW

File: lib/util/support.js [hunk 1]
```diff
@@ -33,6 +42,16 @@ function getParenthesizedSubstrings(string) {
...
 }
 
+/**
+ * Check whether the current browser is a mobile browser
+ * @param {string} userAgent
+ * @returns {boolean}
+ */
+function isMobile(userAgent) {
+  userAgent = userAgent || navigator.userAgent;
+  return /Mobi/.test(userAgent);
```
Comment (manjeshbhargav) [inline lib/util/support.js:52]: Any reason why we are using `Mobi` instead of `Mobile`?

**Review by manjeshbhargav — APPROVED** [review]
@charliesantos LGTM. One nitpick: can you remove the trailing commas? We don't use them in this project, so I want to keep it consistent.

File: lib/util/support.js (hunk 1 above)
Comment (charliesantos) [reply lib/util/support.js:52]: Best practice, as mentioned [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent). Scroll all the way to the bottom. But to summarize, all browsers uses `Mobile`. But opera/presto uses `Mobi`

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,14 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 **Version 1.x will End of Life on September 8th, 2021.** Check [this guide](https://www.twilio.com/docs/video/migrating-1x-2x) to plan your migration to the latest 2.x version. Support for the 1.x version ended on December 4th, 2020.
 
+2.14.1 (In Progress)
+=====================
+
+Bug Fixes
+---------
+
+- Fixed a bug where `isSupported` is returning `true` on certain unsupported mobile browsers. With this release, `isSupported` should now match our list of [supported browsers](https://www.twilio.com/docs/video/javascript#supported-browsers).
```
Comment (manjeshbhargav) [inline CHANGELOG.md:11]:
```suggestion
- Fixed a bug where `isSupported` was returning `true` on certain unsupported mobile browsers. With this release, `isSupported` should now return true only for the [browsers supported by twilio-video.js](https://www.twilio.com/docs/video/javascript#supported-browsers).
```

**Review by manjeshbhargav — APPROVED** [review]

**Pull request merged**